.buildkite/pipeline.yml

```diff
@@ -19,20 +19,22 @@ steps:
 - block: ":rocket: Release !"
   branches: "main"
 
-- command: script/release.sh
+- command: script/bump-version.sh
   if: build.branch == "main"
   label: ":arrow_up_small: Bump & tag version"
 
 - wait
 
 - label: ":github: Publishing artifacts"
   if: build.branch == "main"
+  command: script/release.sh
   plugins:
     - docker#v3.8.0:
-        image: "goreleaser/goreleaser:v0.169.0"
-        command: ["release", "--rm-dist"]
+        image: "golang:1.16.5"
```

Comment:
Is there a way to pull this out into a variable somewhere for when we want to update Go?

Reply:
We could expose that from the agents in the BuildKite UI as an environment variable and use interpolation for this field. 

Or, we could create a dynamic pipeline that reads this value from somewhere and then updates all scripts/yaml files.

Reply:
@fatih Yeah, I don't really have a strong opinion on how, but it would be nice to not have to update too many places somewhere deep in configuration :smile:.

Reply:
I'm more than happy to improve the release pipeline and our scripts once the release process is up and running :)

Reply:
Sound good :+1:.